origins_social_sharing/templates/origins-social-sharing.html.twig

```diff
@@ -0,0 +1,38 @@
+{#
+/**
+* @file
+* Default theme implementation to display a block.
+*
+* @see template_preprocess_block()
+*
+* @ingroup themeable
+*/
+#}
+{{ attach_library('origins_social_sharing/origins_social_sharing_css') }}
+{{ attach_library('origins_social_sharing/origins_social_sharing_js') }}
+<div class="social_sharing">
+  <span class="social_sharing_label">{{ "Share this page"|t }}</span>
+  <span class="social_sharing_buttons" style="line-height: 32px;">
+      <a class="facebook_share_link" target="_blank" href="{{ fb_link }}" rel="nofollow noopener">
+        <span class="facebook_share">
+          <svg focusable="false" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 32 32">
```

Comment:
Can we adapt this to use `https://` by default? We're less likely to get any mixed mode display issues in future that way

Reply:
Good idea, I just used https in the xmlns attribute and it worked fine